src/loopfilter_tmpl_16.rs

```diff
@@ -180,14 +180,13 @@ extern "C" {
     );
 }
 
-pub const DAV1D_X86_CPU_FLAG_AVX512ICL: CpuFlags = 16;
-pub const DAV1D_X86_CPU_FLAG_SSE2: CpuFlags = 1;
-pub const DAV1D_X86_CPU_FLAG_AVX2: CpuFlags = 8;
-pub const DAV1D_X86_CPU_FLAG_SSSE3: CpuFlags = 2;
-pub type CpuFlags = libc::c_uint;
-pub const DAV1D_X86_CPU_FLAG_SLOW_GATHER: CpuFlags = 32;
-pub const DAV1D_X86_CPU_FLAG_SSE41: CpuFlags = 4;
-
+cfg_if! {
```

Comment:
Couldn't these definitions go in the block starting on line 15? If so, I think they should. Same comment for the other template file.

Reply:
I don't think normal `use`s can go in an `extern "C"` block, but I moved them into the `*_init_x86` function like for all the other uses.